OJP/OJP_Booking.xsd

```diff
@@ -222,14 +209,116 @@
 				</xs:annotation>
 			</xs:element>
 			<xs:choice>
-				<xs:annotation>
-					<xs:documentation>This choice is intended for later extension to other mobility services than public transport.</xs:documentation>
-				</xs:annotation>
-				<xs:element name="PublicTransport" type="BookingPTLegStructure"/>
+				<xs:element name="PublicTransport" type="BookingPTLegStructure">
+					<xs:annotation>
+						<xs:documentation>Definition of the journey leg that should be made by public transport. Other mobility services may be added later.</xs:documentation>
+					</xs:annotation>
+				</xs:element>
 			</xs:choice>
+			<xs:group ref="BookableTicketAndServiceInformationGroup"/>
 			<xs:element name="Extension" type="xs:anyType" minOccurs="0"/>
 		</xs:sequence>
 	</xs:complexType>
+	<xs:annotation>
+		<xs:documentation>==================== ticket and service information =====================</xs:documentation>
+	</xs:annotation>
+	<!-- TODO: What is needed from this whole section which has been taken from TRIAS 1.4? Do we already know? -->
+	<xs:group name="BookableTicketAndServiceInformationGroup">
+		<xs:annotation>
+			<xs:documentation>Lists those tickets or surcharges and service characteristics such as the transportation of wheelchairs, prams, luggage etc. or the provision of required child seats or booster seats that are offered in the trips transport company to be booked for the trip.</xs:documentation>
+		</xs:annotation>
+		<xs:sequence>
+			<!-- TODO: Do we already want that? Take TicketStructure from TRIAS? 
+			<xs:element name="BookableTicket" type="TicketStructure" minOccurs="0" maxOccurs="unbounded">
+				<xs:annotation>
+					<xs:documentation>Sequence of bookable tickets or surcharges for this part of the trip. If ticket price is zero: an already purchased/virtual ticket of this type presented to the driver would be accepted.</xs:documentation>
+				</xs:annotation>
+			</xs:element>-->
+			<xs:element name="BookableServiceInformation" type="BookableServiceInformationStructure" minOccurs="0"/>
+		</xs:sequence>
+	</xs:group>
+	<xs:complexType name="BookableServiceInformationStructure">
```

Comment:
Does this structure align with EPIAP?

Reply:
Bookable services from TRIAS align very poorly with EPIAP.
What I've found in EPIAP/NeTEx:
- The possibilty or obligation of reservations is only mentioned for: wheelchairs, bicycles
- Capacities - in a vehicle - are limited to: WheelchairPlaceCapacity, PushchairCapacity, SpecialPlaceCapacity 
- Additionally, we have: PushchairsAllowed, cyclesAllowed, cyclesAllowedWithReservation

Reply:
André: More in TRIAS than in EPIAP. EPIAP only wheelchairs and bicycles. More details there.

Reply:
Stefan: some times only capacity and sometimes more: why? 
André: inconsistent
Malte: I could clear them up.
Stefan: Do we want to go beyond EPIAP, if we can't do it in EPIAP.
Christophe: What not a more general version type+number.
Decision: More general BookableServiceItem with Type  (see new Siri mechanism Counting mechanism in FacilityMonitoring)
Proceed: Matthias includes, example,André takes a look (supply and capacity), Malte checks, then update

Reply:
MonitoredCountingStructure in siri_facility-v2.0.xsd